README.md

```diff
@@ -12,9 +17,15 @@ status](https://www.r-pkg.org/badges/version/ggstackplot)](https://CRAN.R-projec
 experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
 [![codecov](https://codecov.io/gh/kopflab/ggstackplot/branch/main/graph/badge.svg?token=SN0YDIJ6Y6)](https://app.codecov.io/gh/kopflab/ggstackplot)
 [![Documentation](https://img.shields.io/badge/docs-online-green.svg)](https://ggstackplot.kopflab.org/)
+
 <!-- badges: end -->
 
-The goal of ggstackplot is to …
+Have you ever wanted to compare multiple panels of line plots that do
+not share coordinate schemes? These kinds of plots are ubiquitous in the
+Earth sciences, but there does not seem to be a straightforward way to
```

Comment:
how about instead `but there is not an easy way to create them with ggplot facets.` @tacaro - i think we can be a little more assertive